cmd/courier/watch.go

```diff
@@ -0,0 +1,122 @@
+package courier
+
+import (
+	cx "context"
+	"net/http"
+
+	"github.com/gorilla/context"
+	"github.com/rs/cors"
+	"github.com/spf13/cobra"
+	"github.com/urfave/negroni"
+
+	"github.com/ory/graceful"
+	"github.com/ory/kratos/driver"
+	"github.com/ory/kratos/x"
+	"github.com/ory/x/configx"
+	"github.com/ory/x/reqlog"
+	"github.com/ory/x/stringsx"
+)
+
+var watchCmd = &cobra.Command{
+	Use:   "watch",
+	Short: "Starts the ORY Kratos message courier",
+	Run: func(cmd *cobra.Command, args []string) {
+		d := driver.New(cmd.Context(), configx.WithFlags(cmd.Flags()))
+
+		go ServeHealth(d, cmd, args)
```

Comment:
I think having health and metrics servers is honorable :) But we don't really need them IMO. It would make sense if this would handle HTTP API requests. Then, the LB needs to know if the service is up (health) and prometheus would want to know how much latency we have (for example).

So in my opinion, this can be removed!

Reply:
Thanks for the review! 

Yeah, after doing some more reading, I think we can do without liveness probes in this case, and just rely on k8s restarting the pod if the courier crashes. I'll remove this

On the metrics front, you're quite right that latency metrics wouldn't be very relevant here since it isn't handling HTTP requests, but we would like to be able to monitor the go runtime related metrics, particularly since we'd like to be able to measure how well it's performing as we add load and potentially tune things like the number of concurrent goroutines and batch fetch size in response to our observations.

Reply:
I will work on these additional docs today

Reply:
Ok draft of doc updates are in, take a read and don't hesitate to suggest improvements